Source/WebCore/svg/SVGForeignObjectElement.cpp

```diff
@@ -114,8 +119,7 @@ bool SVGForeignObjectElement::rendererIsNeeded(const RenderStyle& style)
     // parentOrShadowHostElement() instead.
     RefPtr ancestor = parentElement();
     while (ancestor && ancestor->isSVGElement()) {
-        // FIXME: [LBSE] Upstream foreignObject support.
-        if (ancestor->renderer() && ancestor->renderer()->isLegacySVGHiddenContainer())
+        if (ancestor->renderer() && (ancestor->renderer()->isSVGHiddenContainer() || ancestor->renderer()->isLegacySVGHiddenContainer()))
```

Comment:
My as well introduce a isSVGHiddenContainerOrLegacySVGHiddenContainer method?

Reply:
Right, however it would only be this place using it -- I only introduced these helpers so far if there is more than one callsite that needs it.